api/HTMLBaseFontElement.json

```diff
@@ -15,10 +15,11 @@
             "version_removed": "79"
           },
           "firefox": {
-            "version_added": false
+            "version_added": "1",
```

Comment:
:) It's fun that we can add this data easily, even though I don't think anyone will start using `<basefont>` now as a result.